From 241296fbb449b9fecc9028aa426d40756171daf0 Mon Sep 17 00:00:00 2001
From: Shen Yong <b00984@freescale.com>
Date: Mon, 8 Sep 2008 19:02:52 +0800
Subject: [PATCH] ENGR00090644 mx51: disable backlight in default config file

due to the PMIC IC bug, disable backlight by default to protect board

Signed-off-by: Kevin Shen <b00984@freescale.com>
---
 arch/arm/configs/imx51_3stack_defconfig |   10 ++--------
 1 files changed, 2 insertions(+), 8 deletions(-)

diff --git a/arch/arm/configs/imx51_3stack_defconfig b/arch/arm/configs/imx51_3stack_defconfig
index 56ee81d..f24e32c 100644
--- a/arch/arm/configs/imx51_3stack_defconfig
+++ b/arch/arm/configs/imx51_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Mon Sep  8 15:50:27 2008
+# Mon Sep  8 17:19:31 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -899,13 +899,7 @@ CONFIG_FB_MXC_EPSON_VGA_SYNC_PANEL=y
 # CONFIG_FB_UVESA is not set
 # CONFIG_FB_S1D13XXX is not set
 # CONFIG_FB_VIRTUAL is not set
-CONFIG_BACKLIGHT_LCD_SUPPORT=y
-CONFIG_LCD_CLASS_DEVICE=y
-# CONFIG_LCD_LTV350QV is not set
-CONFIG_BACKLIGHT_CLASS_DEVICE=y
-# CONFIG_BACKLIGHT_CORGI is not set
-CONFIG_BACKLIGHT_MXC=y
-CONFIG_BACKLIGHT_MXC_MC13892=y
+# CONFIG_BACKLIGHT_LCD_SUPPORT is not set
 
 #
 # Display device support
-- 
1.5.4.4

